pajbot/migration_revisions/db/0016_delete_hsbet_module.py

```diff
@@ -0,0 +1,2 @@
+def up(cursor, bot):
+    cursor.execute("DROP TABLE hsbet_game, hsbet_bet")
```

Comment:
Delete the enum type as well: https://github.com/pajbot/pajbot/blob/7aea8a15fb13cc83295c46e1751965bdf496f6bf/pajbot/migration_revisions/db/0001_initial.py#L115